content/topics/intro-to-relational-databases/_index.md

```diff
@@ -1,15 +1,99 @@
+
 ---
-title: SQL
-ready: True
+title: Introduction to relational database
+ready: true
 ---
 
-## Tutorial
 
-Complete the DataCamp Intro and Intermediate SQL courses:
+### INTRODUCTION TO RELATIONAL DATABASE (SQL)
+
+
+#### What is Relational DataBase? 
+
+Learn more here:[What is RDBMS](https://www.youtube.com/watch?v=h8IWmmxIyS0)
+
+- Introduction to SQL relational database [visit](https://www.mikedane.com/databases/sql/)
+
+- Basic SQL queries: [See these](https://www.mikedane.com/databases/sql/basic-queries/)
+
+
+#### LESSONS
+
+##### Lesson1:
+
+- Keys / Data and Tables:
+> [check these out](https://www.mikedane.com/databases/sql/tables-keys/)
+
+##### Lesson2:
+
+- Basics/Elements of SQL 
+> [check these out](https://www.mikedane.com/databases/sql/sql-basics/)
+
+##### Lesson3:
+  
+- SQL-Installation:
+   
+    > [windows](https://www.mikedane.com/databases/sql/mysql-windows-installation/)
+
+    > [MAC / Linux/ Ubuntu](https://www.mikedane.com/databases/sql/mysql-mac-installation/)
+
+##### Lesson4:
+  
+- Creating Tables
+> [Click here](https://www.mikedane.com/databases/sql/creating-tables/)
+
+##### Lesson5:
+  
+- Inserting Data
+> [check it out](https://www.mikedane.com/databases/sql/inserting-data/)
+  
+**Lesson6:**
+  
+- Constraints
+> [check it out](https://www.mikedane.com/databases/sql/constraints/)  
+
+##### Lesson7:
+
+- Update & Delete Data
+> [check it out](https://www.mikedane.com/databases/sql/basic-queries/)
+
+##### Building APIs:
+
+- Nodejs API with SQL:
+> [check it out](https://www.youtube.com/watch?v=EN6Dx22cPRI&t=20s) or [also check this out](https://www.youtube.com/watch?v=hqkyIKIeFe0)
+
+> REST API using Python and Flask | Flask-Restful: 
+> [check it out & start coding](https://www.youtube.com/watch?v=s_ht4AKnWZg)
+
+#### Understanding difference between Realtion & Non relational database:
+
+##### Structure vs Non structured query language: 
+> Learn more about SQL & NoSql: [Click here!](https://www.youtube.com/watch?v=ZS_kXvOeQ5Y)
+> e.g. 
+                ![alt text](https://miro.medium.com/max/1512/1*LEksJP5OtS8GEBdd2Jy4WQ.png)
+
+**Best practices**  
+```
+*(N.B)* 
+If your using VIM or VI as your code Editor...
 
-- [Intro to SQL for data science](https://www.datacamp.com/courses/intro-to-sql-for-data-science)
-- [Intermediate SQL](https://www.datacamp.com/courses/intermediate-sql)
+Always do not forget to include your .gitignore file
```

Comment:
always try to not use double negatives because its seldom not confusing.

To put it cleanly: Don't use double negatives, it is confusing